app/components/google-jsapi.js

```diff
@@ -1,15 +1,16 @@
-import Ember from 'ember';
+import Component from '@ember/component';
+import $ from 'jquery';
 
-export default Ember.Component.extend({
+export default Component.extend({
     tagName: '',
 
     didInsertElement() {
-        Ember.$.getScript('https://www.google.com/jsapi', function() {
+        $.getScript('https://www.google.com/jsapi', function() {
```

Comment:
hm, I guess these will be short lived anyway, as we'll want to use something fastboot-compatible.